static/js/stream_color.js

```diff
@@ -128,7 +128,7 @@ exports.sidebar_popover_colorpicker_options_full = {
     showInput: true,
     flat: true,
     cancelText: "",
-    chooseText: "choose",
+    chooseText: i18n.t("choose"),
```

Comment:
Ahh, I see.

It's defined in `zproject/settings.py`.  `static/js/src/main.js` is, via webpack, transformed into `bundle.js`, which you can see is appended at the very bottom of the list.  

```
if PIPELINE_ENABLED:                                                                  
    # This is also done in test_settings.py, see comment there..                      
    JS_SPECS['app']['source_filenames'].append('js/bundle.js')                        
```

So I think we could change that line to insert it earlier in the list (e.g. just before `js/feature_flags.js`) and see if that helps...